apps/condo/domains/organization/schema/OrganizationEmployee.test.js

```diff
@@ -1,267 +1,239 @@
 
-    test('anonymous: no access to count', async () => {
-        const client = await makeClient()
-        const { data, errors } = await OrganizationEmployee.count(client, {}, { raw: true })
-        expect(errors[0]).toMatchObject({
-            'data': { 'target': '_allOrganizationEmployeesMeta', 'type': 'query' },
-            'message': 'You do not have access to this resource',
-            'name': 'AccessDeniedError',
-            'path': ['meta', 'count'],
+        test('cannot without granted "canManageEmployees" permission', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [organization] = await createTestOrganization(admin)
+            const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                canManageEmployees: false,
+            })
+            const notManagerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+            await createTestOrganizationEmployee(admin, organization, notManagerUserClient.user, role)
+            const { user } = await makeClientWithNewRegisteredAndLoggedInUser()
+            try {
+                await createTestOrganizationEmployee(notManagerUserClient, organization, user, role)
+            } catch (e) {
+                expect(e.errors[0]).toMatchObject({
+                    'message': 'You do not have access to this resource',
+                    'name': 'AccessDeniedError',
+                    'path': ['obj'],
+                })
+                expect(e.data).toEqual({ 'obj': null })
+            }
         })
-        expect(data).toEqual({ meta: { count: null } })
-    })
 
-    test('user: allow to getAll only for employee', async () => {
-        const client1 = await makeClientWithRegisteredOrganization()
-        const client2 = await makeClientWithRegisteredOrganization()
-        const objs1 = await OrganizationEmployee.getAll(client1, {})
-        const objs2 = await OrganizationEmployee.getAll(client2, {})
-        expect(objs1).toHaveLength(1)
-        expect(objs2).toHaveLength(1)
-        expect(objs1[0].id).not.toEqual(objs2[0].id)
-    })
-
-    test('user: allow to count only for employee', async () => {
-        const client1 = await makeClientWithRegisteredOrganization()
-        const client2 = await makeClientWithRegisteredOrganization()
-        const count1 = await OrganizationEmployee.count(client1, {})
-        const count2 = await OrganizationEmployee.count(client2, {})
-        expect(count1).toEqual(1)
-        expect(count2).toEqual(1)
-    })
-
-    test('admin: allow to getAll', async () => {
-        const client1 = await makeClientWithRegisteredOrganization()
-        await addAdminAccess(client1.user)
-        const objs1 = await OrganizationEmployee.getAll(client1, {})
-        expect(objs1.length).toBeGreaterThan(1)
-    })
+        test.skip('can with granted "canManageEmployees" permission', async () => {
```

Comment:
why skip?

Reply:
There is a comment inside of test:
```
// Exception will be thrown, because of `OrganizationEmployee.fields.user.access.create: userIsAdmin`
// It's impossible to test from OrganizationEmployee perspective with such restriction on `OrganizationEmployee.user` field
```

Reply:
Found solution in `f9abaf2`